src/pages/HomePage/pages/LifeCenter/pages/ViewLifeCenter.tsx

```diff
@@ -0,0 +1,72 @@
+import { useFetch } from "@/CustomHooks/useFetch";
+import { api } from "@/utils/api/apiCalls";
+import { decodeQuery } from "@/pages/HomePage/utils";
+import { useParams } from "react-router-dom";
+import { SoulsWon } from "../components/SoulsWon";
+import PageOutline from "@/pages/HomePage/Components/PageOutline";
+import VisitorIcon from "@/assets/sidebar/VisitorIcon";
+import { useEffect, useState } from "react";
+import { SoulsWonType } from "@/utils";
+
+export function ViewLifeCenter() {
+  const { id } = useParams();
+
+  const LCId = decodeQuery(String(id));
+  const { data } = useFetch(api.fetch.fetchLifeCenterById, {
```

Comment:
this is reactive and so can be used directly in the rendered potion
meaning there is no need for the useState